src/Arcus.Templates.WebApi/Arcus.Templates.WebApi.csproj

```diff
@@ -60,8 +61,8 @@
     <PackageReference Include="Arcus.WebApi.Correlation" Version="20200121.10.0-preview" Condition="'$(ExcludeCorrelation)' == 'false'" />
     <PackageReference Include="Arcus.WebApi.Logging" Version="0.3.0" />
     <PackageReference Include="Arcus.WebApi.Security.Authentication" Version="0.3.0" Condition="'$(Auth)' == 'true'" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="4.0.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="4.0.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="4.0.1" Condition="'$(ExcludeOpenApi)' == 'false'" />
```

Comment:
We should do the same for `GenerateDocumentationFile` here above

Reply:
Ah, yes. Thanks.